### DISCUSSION TITLE
what about ASR for onnx?

### DISCUSSION THREAD

**Discussion by hqgitbub** [PaddlePaddle/PaddleSpeech#2651] [Q&A]
The system just support TTS for onnx.
What about ASR for onnx?

Also I did not find any discussion about it ?

What about your plan?

**Comment by zh794390558** [comment]
You can try to do it. And now we do not have this plan.

**Reply by hqgitbub** [reply]
Pity! 
Paddlespeech left a deep impression on me. 
It's a very good project.
Hope your next step about this plan.

**Reply by hqgitbub** [reply]
Also thanks for your replay.

**Comment by SmileGoat** [comment]
https://github.com/PaddlePaddle/PaddleSpeech/tree/develop/speechx/examples/ds2_ol/onnx

**Reply by hqgitbub** [reply]
I read the topic before. But not get point.
My test model is https://paddlespeech.bj.bcebos.com/s2t/wenetspeech/asr1_conformer_wenetspeech_ckpt_0.1.1.model.tar.gz .
You remind me.
I did not try it locally, but I can try it lately.

But the other big problem is to adatp the pre and post process.
Also thanks for you replay.

**Reply by iftaken** [reply]
> I read the topic before. But not get point. My test model is https://paddlespeech.bj.bcebos.com/s2t/wenetspeech/asr1_conformer_wenetspeech_ckpt_0.1.1.model.tar.gz . You remind me. I did not try it locally, but I can try it lately.
> 
> But the other big problem is to adatp the pre and post process. Also thanks for you replay.

It's a good advise for PaddleSpeech. 
May i ask normally what kind of  application scenarios do you use with onnx model?

more advices are welcome
thanks
